main.go

```diff
@@ -2,30 +2,48 @@ package main
 
 import "fmt"
 
+var restaurant string = "Jo's coffee shop!"
+
 type List struct {
 	productID   int
 	productName string
 	price       int
 }
 
 func getList() []List {
-	l1 := List{
-		productID:   1,
-		productName: "Latte",
-		price:       150,
+	return []List{
+		{
+			productID:   1,
+			productName: "Latte",
+			price:       150,
+		},
+		{
+			productID:   2,
+			productName: "Filter coffee",
+			price:       100,
+		},
 	}
-	l2 := List{
-		productID:   2,
-		productName: "Filter coffee",
-		price:       100,
+}
+func handleCommand(command string) {
+	switch command {
+	case "menu":
+		fmt.Println("Menu：")
+		lists := getList()
+		for _, product := range lists {
+			fmt.Printf("%v. %v, $%v\n", product.productID, product.productName, product.price)
```

Comment:
Using `%v` for everything works correctly, but a cleaner implementation would be to use the correct placeholders for the respective data types, e. g. `%d`, `%s`.